docs/docusaurus/versioned_docs/version-1.6.X/orc8r/upgrade_1_5.md

```diff
@@ -66,7 +66,7 @@ This process prepares your cluster for Orc8r-NMS DB unification. (In release 1.5
 *Paste in shell prompt:*
 
 ```bash
-wget https://raw.githubusercontent.com/magma/magma/master/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
+wget https://raw.githubusercontent.com/magma/magma/v1.5/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
```

Comment:
Typo here, it looks like this should be pointing to v1.6 branch

Reply:
That was intentional because, this is a script to upgrade to version 1.5. I can change this, but this means we would have to adapt the link with every release.

Reply:
@andreilee I think your reply is necessary to unblock

Reply:
@thmsschmitt hm you're right. My mistake.